packages/docs/docs/components/nodes.mdx

```diff
@@ -15,10 +15,11 @@ import texSource from '!!raw-loader!@motion-canvas/examples/src/scenes/tex';
 
 ## A note on tweening LaTeX
 
-Because we use a canvas renderer, we're rendering the LaTeX to an image instead
-using a full graphics backend to render the TeX. This means that for all intents
-and purposes, LaTeX should be treated as if it were an image rather than more
-flexible TeX rendering like you may see in other software.
+Because we use a canvas renderer, we're rendering the LaTeX to an image.
+This means we're not using a full graphics backend to render the TeX. Thus,
+for all intents using a full graphics backend to render the TeX. This means
```

Comment:
> This means we're not using a full graphics backend to render the TeX. Thus,
for all intents using a full graphics backend to render the TeX.

What does this mean?